docs/en/reference/generating-migrations.rst

```diff
@@ -217,7 +217,7 @@ It simply takes a schema object to its constructor and returns it from ``createS
 
 By default the Doctrine Migrations command line tool will only add the diff command if the ORM is present.
 Without the ORM, you'll have to add the diff command to your console application manually, passing in your schema
-provider implementation to the diff command's constructor. Take a look at the :ref:`Custom Integration <custom-integration>`
+provider implementation to the DependencyFactory. Take a look at the :ref:`Custom Integration <custom-integration>`
```

Comment:
More like "passing in your dependency factory implementation to the diff command's constructor."

Reply:
Thanks!

For me the part I did not understand first, was: I need to call `setDefinition(SchemaProvider::class, static fn () => $schemaProvider)` on the dependency factory. Everything else is already sufficiently explained here: https://www.doctrine-project.org/projects/doctrine-migrations/en/3.3/reference/custom-integration.html which is referenced in the sentence below "Take al look..."

What do you think about:
Without the ORM, you'll have to add the diff command to your console application manually and set your custom schema provider to the dependency factory, which will be passed to the the diff command's constructor. Take a look ...

Reply:
I thinks that sounds comprehensible to me.

Reply:
thanks! updated